module/PowerShellEditorServices/Commands/Private/BuiltInCommands.ps1

```diff
@@ -3,7 +3,7 @@ Register-EditorCommand `
     -DisplayName 'Open Editor Profile' `
     -SuppressOutput `
     -ScriptBlock {
-        param([Microsoft.PowerShell.EditorServices.Services.PowerShellContext.EditorContext]$context)
+        param([Microsoft.PowerShell.EditorServices.Extensions.EditorContext, Microsoft.PowerShell.EditorServices]$context)
```

Comment:
Does this syntax work with 5.1? I vaguely remember you running into issues where you needed the full qualification in 5.1.  Or was it that the full qualification didn't work but this did?

Reply:
Ah good thinking! Now that you mention it, I remember a similar issue (with PSRL type invocation).

But I just tested it and this works in 5.1:

```
PS C:\Users\Robert Holt\Documents\Dev\sandbox> [microsoft.powershell.editorservices.extensions.editorcommand, microsoft.powershell.editorservices]


IsPublic IsSerial Name                                     BaseType
-------- -------- ----                                     --------
True     False    EditorCommand                            System.Object
```